actix-http/Cargo.toml

```
@@ -15,7 +15,7 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [package.metadata.docs.rs]
-features = ["openssl", "rustls", "failure", "compress", "secure-cookies"]
+features = ["openssl", "rustls", "failure", "compress", "secure-cookies","actix"]
```

feature name is actors or actix ?

---

Yeah, should be `actors`.

```suggestion
features = ["openssl", "rustls", "failure", "compress", "secure-cookies","actors"]
```

---

Oh, right, sorry, it was late night fix. I have pushed with few more documentation changes.